tests/network/connectivity/conftest.py

```diff
@@ -59,14 +59,14 @@ def nncp_ovs_bridge_device_worker_1_source(
         nncp_name=f"ovs-bridge-{name_prefix(worker_node1.name)}",
         interface_name=bridge_device_name,
         node_selector=get_node_selector_dict(node_selector=worker_node1.hostname),
-        ports=[nodes_available_nics[worker_node1.name][-1]],
+        ports=[hosts_common_available_ports[-1]],
     ) as br:
         yield br
 
 
 @pytest.fixture(scope="class")
 def nncp_linux_bridge_device_worker_2_destination(
```

Comment:
in the orig code, `worker_node2` was used. with the new change, this is no nger the case so the fixture name needs to be updated

Reply:
Oh yeah thanks for pointing that @yossisegev , i have reverted back to the original code.